services/core/shared/core/compat/sdk_v2/blocks.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const coreApi = require('./coreApi');
+const {
+	getUnixTime,
+	getBlockchainTime,
+	validateTimestamp,
+} = require('../common');
+
+let currentHeight = 0;
+const considerFinalHeight = 202;
```

Comment:
Consider referring to the config property instead.

Reply:
This value has nothing to do with the config value. We can cache as many blocks as we want, but this constant is exclusive to pre-v3 blockchains.